pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -278,11 +280,11 @@ func (state PipelineRunState) getNextTasks(candidateTasks sets.String) []*Resolv
 }
 
 // IsStopping returns true if the PipelineRun won't be scheduling any new Task because
-// at least one task already failed or was cancelled in the specified dag
+// at least one task already failed (with onError: stopAndFail) or was cancelled in the specified dag
```

Comment:
nit: "stopAndFail" is the default here this sounds like users have to set OnError?
Will leave this up to your decision.